src/NetPanzer/Views/MainMenu/MenuTemplateView.cpp

```diff
@@ -273,8 +273,8 @@ void MenuTemplateView::doDrawAlt(Surface &viewArea, Surface &clientArea) //help
 	// When ingame, tint the game into gray
 
         //clientArea.BltRoundRect(getClientRect(), 10, Palette::gray256.getColorArray());
-        //clientArea.FillRoundRect(getClientRect(), 10, Color::white); // esc in game
-             //clientArea.RoundRect(MenuRect, 10, Color::gray);
+        clientArea.FillRoundRect(getClientRect(), 10, Color::white); // esc in game
+             clientArea.RoundRect(MenuRect, 10, Color::gray);
```

Comment:
note - file needs reformatted... I just made the minimal changes as possible to make review easier.